Add-ons/Umbraco-Deploy/Deployments/index.md

```diff
@@ -0,0 +1,39 @@
+---
+versionFrom: 8.0.0
+---
+
+# Deployment
+Umbraco Deploy uses a deployment model that relies on Git and Umbraco Deploy core technology to move your changes from one environment to another. Umbraco Deploy uses a classic "left to right" deployment model, meaning that changes are first made in he Development or local environment and then deployed to the production environment.
+
+__Note:__ If your project contains a Staging environment, deployments will be made from Development to Staging and then from Staging to Live.
+
+![Left to right model](images/left-to-right.png)
+
+## Deployment Approach
+
+Umbraco Deploy uses a two-part deployment approach where we keep meta data (Document types, templates, etc) and content (Content nodes and Media) as separate parts of a deployment. In order to be able to distinguish between the two types of deployments we use the term *transfer* for content and media deployments and the term *deploy* for meta data deployments.
+In summary:
+
+1. Meta data such as Document Types, Templates, Forms, Views and config files are stored in a git repository and are **deployed** between environments using either a git client or using the command prompt.
```

Comment:
Again, this is Cloud specific,  You'll need to reference a build and deployment pipeline here.  This might be triggered by a git commit, but it doesn't have to be.